_layouts/default.html

```diff
@@ -16,7 +16,8 @@
     {%- include header.html %}
 
     <!-- Content -->
-    <div class="container mt-5">
+    <div class="container mt-5" {% if site.rtl_langs contains page.lang %}dir="rtl" align="right"{% endif %}
```

Comment:
I think it's also worth adding `"rtl"` class to this div, so that we can conditionally change CSS of some inner elements (eg, the border location of the blockquote).

Reply:
Happy that you find it useful :) . I have to clarify one important fact: I'm by no means dominant on HTML, SCSS, and I'm afraid, even after going through Jekyll's documentation several times, I haven't fully comprehend its philosophy. So please take my response with a grain of salt, i.e., modify it as you feel it's better.

Coming back to your comment, I actually don't know how to add it to `div` here. If you provide me with some hints, I would be happy to work on it.

Reply:
Hi, Thanks for this great idea. 

It is a common practice ([+](https://stackoverflow.com/a/47376504/4201765)) to add `dir=rtl` in a separate div. So, maybe it is better to add it like this:
```html
<div {% if site.rtl_langs contains page.lang %} dir="rtl" endif %>
Stuff
</div>
```

Also, I see that you have added `align=right`. Why do you think that is necessary? I haven't seen it anywhere.

Reply:
Hi, 
I was not aware of this practice, but it seems to work on my local machine. 

`align=right` is to decouple the text of each post from the body of the page which have some use cases. E.g., one might want to have the header and navigation in, say English, but the posts in a rtl script. An example post is [here](https://arashgmn.github.io/blog/2022/dialogue-prerequisites/).